File: Python/Extra_Files/listfunc.go
package main

import (
"fmt"
"sort"
)

func main(){

var markList=[] int {78,90,100,96,83,95}
fmt.Println("The marks are:",markList)

markList=append(markList,55)
fmt.Println("After adding new marks:",markList)

var i int=2
markList = append(markList[:i], markList[i+1:]...)
fmt.Println("After removing the marks at the 2nd position:",markList)

sort.Ints(markList)
fmt.Println("After sorting the marks in the given list:",markList)
}

File: Python/Extra_Files/listfunc.js
markList=[78,17,90,90,55,95,83,95]
console.log("The marks are:"+markList+"")

markPos=markList.indexOf(9)
console.log("Index position of mark 90:"+ markPos)


markList.push(54)
console.log("After adding new marks:"+markList+"")

markList.splice(2, 0, 98)
console.log("After inserting 98 at the 2nd position:"+markList+"")

markIndex=markList.indexOf(90)
if(markIndex != -1) {
    markList.splice(markIndex, 1);
}
console.log("After removing first 90 from the list:"+markList+"")

markList.reverse()
console.log("After reversing the marks in the given list:"+markList+"")

nameList=["Marcus","James","Andrew","Bond"]
console.log("Before Sorting:"+nameList)
nameList.sort()
console.log("After Sorting:"+nameList)